lib/rex/post/meterpreter/ui/console/command_dispatcher/extapi/clipboard.rb

```diff
@@ -431,7 +461,11 @@ def parse_dump(dump, get_images, get_files, download_path)
             print_line("Remote Path : #{f[:name]}")
             print_line("File size   : #{f[:size]} bytes")
             if get_files
-              unless download_file(loot_dir, f[:name])
+              download_status, attempt = download_file(loot_dir, f[:name],force_overwrite)
+              #if once set to true, leave it true
+              overwrite_attempt = overwrite_attempt ? true : attempt
```

Comment:
I think the following would work, but would need to double-check:
```suggestion
              overwrite_attempt ||= attempt
```